systemd/ceph-disk@.service

```diff
@@ -6,5 +6,6 @@ Wants=local-fs.target
 [Service]
 Type=oneshot
 KillMode=none
-ExecStart=/bin/sh -c 'timeout 120 flock /var/lock/ceph-disk-$(basename %f) /usr/sbin/ceph-disk --verbose --log-stdout trigger --sync %f'
+Environment=CEPH_DISK_TIMEOUT=300
+ExecStart=/bin/sh -c 'timeout $${CEPH_DISK_TIMEOUT} flock /var/lock/ceph-disk-$(basename %f) /usr/sbin/ceph-disk --verbose --log-stdout trigger --sync %f'
```

Comment:
can we just pass `$CEPH_DISK_TIMEOUT` here? to let systemd instead of `sh` to do the expansion.